mir-ci/mir_ci/robot_resources/screencopy.resource

```diff
@@ -8,93 +9,161 @@ Library             ../robot_libraries/WaylandHid.py
 *** Keywords ***
 Get Center Of ${region}
     [Documentation]    Get the center point of a region.
+    ...
+    ...    Arguments:
+    ...    - ${region}: Rectangular region represented by a dictionary
+    ...      of integer values for "left", "right", "top", and "bottom" keys.
+    ...
+    ...    Return:
+    ...    Center of the region, as a tuple (x, y) of integers.
     ${point}=    Evaluate
-    ...    {"x": (${region}[left] + ${region}[right]) / 2, "y": (${region}[top] + ${region}[bottom]) / 2}
+    ...    (int((${region}[left] + ${region}[right]) / 2), int((${region}[top] + ${region}[bottom]) / 2))
     RETURN    ${point}
 
 Displace ${point} By (${x}, ${y})
-    [Documentation]    Shift a point by the specified displacements along the x and y axes.
-    ${point}=    Evaluate    {"x": ${point}[x] + ${x}, "y": ${point}[y] + ${y}}
+    [Documentation]    Shift a point by the specified displacements along
+    ...    the x and y axes.
+    ...
+    ...    Arguments:
+    ...    - ${point}: Point to be displaced, as a tuple (x, y).
+    ...    - ${x}: Displacement along the x-axis.
+    ...    - ${y}: Displacement along the y-axis.
+    ...
+    ...    Return:
+    ...    Displaced point, as a tuple (x, y) of integers.
+    ${point}=    Evaluate    (int(${point}[0] + ${x}), int(${point}[1] + ${y}))
     RETURN    ${point}
 
-Move Pointer To ${destination}
-    [Documentation]    Move the pointer to a position or image template.
+Move Pointer To (${x}, ${y})
+    [Documentation]    Move the pointer to an absolute position.
+    ...
+    ...    Arguments:
+    ...    - ${x}: Integer absolute x-coordinate to move the pointer to.
+    ...    - ${y}: Integer absolute y-coordinate to move the pointer to.
+    WaylandHid.Move Pointer To Absolute    ${x}    ${y}
```

Comment:
If we introduce another Hid library, this would be easier:

```suggestion
    Move_Pointer_To_Absolute    ${x}    ${y}
```

Reply:
Some of these will conflict with keywords with the same name in the resource file. Instead of dropping `WaylandHid`, I changed it to `HID` and imported the library `AS    HID`:
```robot
Library             ../robot_libraries/WaylandHid.py    AS    HID
```
WDYT?